stglibs/pinger.lib/include/stg/pinger.h

```diff
@@ -19,6 +19,7 @@
 #endif
 
 #if defined (FREE_BSD) || defined (FREE_BSD5) || defined(DARWIN)
+#include <sys/types.h>
```

Comment:
I don't think this fix applies for Darwin platform. Please limit it to FreeBSD and likely to FreeBSD5 and later only - there was no such problem on FreeBSD4 with gcc.